Health App/Home/HealthConcernsListVC.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  HealthConcernsListVC.swift
+//  Health App
+//
+//  Created by Koushik Dutta on 07/10/17.
+//  Copyright © 2017 Virtual Elf LLC. All rights reserved.
+//
+
+import UIKit
+import DataLayer
+
+class HealthConcernsListVC: UIViewController {
+
+    var healthConcerns: [HealthConcern] = []
+    var inControlStatus: [HealthConcern] = []
+    var notInControlStatus: [HealthConcern] = []
+    var resolvedStatus: [HealthConcern] = []
+    @IBOutlet weak var healthConcernsTableView: UITableView!
+    @IBOutlet weak var addNewHealthConcernBarButton: UIBarButtonItem!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        healthConcerns = DatabaseManager.shared.fetchHealthConcerns()
+        divideBasedOnStatus()
+        healthConcernsTableView.reloadData()
+    }
+    
+    func divideBasedOnStatus(){
+        inControlStatus.removeAll()
+        notInControlStatus.removeAll()
+        resolvedStatus.removeAll()
+        for healthConcern in healthConcerns{
```

Comment:
use NSFetchRequest

`let fetchRequest : NSFetchRequest<HealthConcern> = HealthConcern.fetchRequest()
            fetchRequest.predicate = NSPredicate(format: "status == %@", "In Control")`

Reply:
Load it from DatabaseManager